issues.md

```diff
@@ -1,156 +1,150 @@
+3. **Do the research:** Most issues need some investigation before work can start.
 
-        - If they agree for you to take it, we'll assign you to it and remove them.
-        - If they agree for you to take it and some work has been done already, or the agreement is to help them out with it, we'll assign you to it alongside them.
+   - Scope out the work that needs to be done. If you need to ask questions, ask for help in the [MDN Web Docs chat room](https://chat.mozilla.org/#/room/#mdn:mozilla.org) on [Matrix](https://wiki.mozilla.org/Matrix).
+   - If the issue is well-described, and the work is pretty obvious, go ahead and do it.
+   - If the issue is not well-described, and/or you are not sure what is needed, feel free to @mention the poster and ask for more information.
+   - If you are not still sure who to ask, ask for help in the [MDN Web Docs chat room](https://chat.mozilla.org/#/room/#mdn:mozilla.org) on [Matrix](https://wiki.mozilla.org/Matrix).
 
-     2. If it was less than one week ago, then have some patience, and give them a chance to work on it.
+4. **Make the changes:** Fork and branch the repository. Do your work and open a [pull request](/en-US/docs/MDN/Community/Pull_requests) in the repository. Comment in the issue and ask for a review of your pull request.
 
-2. If the issue has been marked as complete but needing a review, and you want to review it, @mention them in the comments and say you'll review it.
+   If you no longer have the time to work on the assigned issue, let the team know in a comment so that the issue can be assigned to someone else.
 
-## When you've been assigned to an issue
+5. After your pull request has been reviewed and merged, you can mark the linked issue as closed. If you opened the pull request with `Fixes #<issue>` verbiage, the issue will be closed automatically when the pull request is merged.
 
-1. Scope out the remainder of the work that needs to be done.
+### Fixing issues yourself
 
-   - If the issue is well-described, and the work is pretty obvious, go ahead and do it.
-   - If the issue is not well-described, and/or you are not sure what is needed, feel free to @mention the poster and ask for more information.
-   - If you are not still sure who to ask, ask for help in the [MDN Web Docs chat room](https://chat.mozilla.org/#/room/#mdn:mozilla.org) on [Matrix](https://wiki.mozilla.org/Matrix).
+If you spot a bug — whether it's a problem with site infrastructure or an error in documentation — you can try to fix it yourself.
 
-2. Once you think you've fixed an issue, ask for a review in the comments.
-3. Once an issue has been successfully reviewed and comments answered, you can mark it as closed.
-4. If you no longer have time to work on an issue, let us know in a comment so we can assign it someone else.
+[Report the problem](#guidelines_for_reporting_an_issue) as described earlier.
```

Comment:
Nice, we should also hint that you should assign yourself to the issue or comment / indicate that you want to fix it / start working on it.

Reply:
I also added that after opening the issue, you can indicate your intent to work on the issue, but wait for it to be triaged. After triaging, assign yourself to the issue to start working on it.